### ISSUE TITLE
node-red-node-email not getting mails (Office365, IMAP)/Documentation for POP wrong?

### ISSUE THREAD

**Issue by koshisan** [node-red/node-red-nodes#792]
### Which node are you reporting an issue on?

node-red-node-email 

### What are the steps to reproduce?

Just put a node-red-node-email in the flow, add a debug message object and configure credentials. Creteria is set to none. To my understanding of the documentation that should read the latest mail on every connect/at least once. 

### What happens?

The connection to the server seems to be successful, at least that is what the state in the gui tells my, however, the debug message is never triggered

### What do you expect to happen?

To my understanding of the documentation it should either trigger the debug message every time with the latest mail or at least once. 

Bonus information: The documentation says "Note: this node only gets the most recent single email from the inbox, so set the repeat (polling) time appropriately." - this is also not true: When switchting to POP3 for testing purposes, the node started to get _all_ of my ~20.000 mails in my inbox...

### Please tell us about your environment:

- [ ] Node-RED version: 1.2.9
- [ ] node.js version: v12.21.0
- [ ] npm version: 6.14.11
- [ ] Platform/OS: Ubuntu 18 LTS
- [ ] Browser: various/unrelated

**Comment by dceejay** [comment]
Which version of the email node do you have - I think the latest docs do say iMAP should also fetch all the emails.
Does the status indicate any error at all with IMAP ? (eg with gmail you need to use an application password not your real password which catches a lot of people out for example). I don't have a 365 server to try against.

**Comment by koshisan** [comment]
I just installed it today, so it is the most recent version...
The information with the "only one mail"-thing is from here: https://flows.nodered.org/node/node-red-node-email
There isn't any status indicated, however, I tried to enter a wrong password to see if the authentification passes and with the wrong password an authentication error is clearly visible - so I guess the problem is post authentication.

However, if it tries to read all mails it is most probably a timeout issue with the amount of mails in my inbox. I did try to debug with Firebird and it took quite a while to get through the message list...

I will try using an empty folder and give an update again!
